### ISSUE TITLE
[Suggestion] Final Star changes

### ISSUE THREAD

**Issue by Firesparq** [LatvianModder/Project-EX#37]
Instead of having the final star duplicate EMCable items (which at a point where you have stupid amounts of EMC anyways). Instead make it to duplicate a placed block and not limit it to EMCable items. That way, you're rewarded for making it to end game.

**Comment by LatvianModder** [comment]
There is a config for that, but it doesnt consume star

**Comment by Firesparq** [comment]
Yes, I'm aware of that. Conceptually though, you play through the projectex progression, only to have almost infinite emc and your reward? infinite emc. Why not have something that adds a bit of spice and allow duplication of non-emcable items, but it costs a huge amount of emc; the final star.
